src/_factories.ts

```diff
@@ -94,10 +47,14 @@ function isInstance(value: any): value is Instance {
 }
 
 function resolveFactory(type: 'action', definition: ActionDefinition, resolveMid: ResolveMid): Promise<ActionFactory>;
```

Comment:
Although I suspect this is internal, some JSDoc would help in understanding this better.